owtf.py

```diff
@@ -255,4 +255,9 @@ def main(args):
 
 
 if __name__ == "__main__":
+
+    if os.path.dirname(os.path.abspath(sys.argv[0])) == os.getcwd():
```

Comment:
If executed file folder is substring of current working directory, it is so then we can say that OWTF is ran from its directory or subdirectory.